### DISCUSSION TITLE
modularizeImports with react-icons?

### DISCUSSION THREAD

**Discussion by stx-chris** [vercel/next.js#52313] [App Router]
Anybody having luck applying `modularizeImports` to `react-icons`? Right now (13.4.8) it doesn't tree-shake out of the box and tried several configurations, but no luck. I'd appreciate any hint on reducing bundle size with `react-icons`. Thx!

**Comment by imageck** [comment]
Unfortunately, not many [examples](https://github.com/search?q=%22modularizeImports%22+AND+%22react-icons%2F%22+path%3Anext.config&type=code) out there and the package itself doesn't seem to provide a single barrel file. 

AFAIK, `modularizeImport` is just a transform. It doesn't require tree-shaking as your import declarations are converted automatically.

**Comment by paulwongx** [comment]
Any update on this?
